config.xsd

```diff
@@ -320,13 +320,13 @@
             <xs:element name="MixedArrayAssignment" type="IssueHandlerType" minOccurs="0" />
             <xs:element name="MixedArrayOffset" type="IssueHandlerType" minOccurs="0" />
             <xs:element name="MixedArrayTypeCoercion" type="IssueHandlerType" minOccurs="0" />
-            <xs:element name="MixedAssignment" type="IssueHandlerType" minOccurs="0" />
```

Comment:
It should remain here, but be marked as deprecated. See https://github.com/orklah/psalm/blob/24ae96b373cb44186c58360543dcf0987436df0c/config.xsd#L20-L25 for example

Reply:
As discussed, it doesn't seem like anyone references it in `psalm.xml`

Reply:
Github search shows 97 entries, including some popular repos like `doctrine/collections`: https://github.com/search?q=path%3Apsalm.xml+language%3AXML+MixedAssignment&type=code